pkg/rpm/google-cloud-ops-agent.spec

```diff
@@ -46,6 +47,7 @@ CODE_VERSION=%{version} BUILD_DISTRO=${build_distro#.} DESTDIR="%{buildroot}" ./
 %{_prefix}/libexec/google_cloud_ops_agent_engine
 %{_unitdir}/%{name}*
 %{_unitdir}-preset/*-%{name}*
+%{_logrotatedir}/google-cloud-ops-agent
```

Comment:
Should this be marked as "config"? I assume users will potentially want to edit this…